test/run_all

```diff
@@ -70,7 +70,8 @@ class TestRunner(object):
         self._outputq = Queue()
 
     def _find_tests(self):
-        for test in glob.glob("*.t") + glob.glob("*.t.exe"):
+        for test in glob.glob("*.t") + glob.glob("*.t.exe") \
```

Comment:
is the first pattern still needed? Are there any remaining `*.t` files?

Reply:
Yes, the `*.t` is currently still needed as the cpp unit tests (ending `*.t.cpp`) are created as targets with the ending `*.t`. 
See, this lines where `src_FILE` is set based on line 21:
https://github.com/GothenburgBitFactory/taskwarrior/blob/eaef05ee95934e74db0f19dd1470c8424b17962c/test/CMakeLists.txt#L30-L36

Currently the `*t.exe` is not in use. 
Same applies for https://github.com/GothenburgBitFactory/taskwarrior/pull/3407#discussion_r1581716066.